sparkly/utils.py

```diff
@@ -59,6 +64,89 @@ def absolute_path(file_path, *rel_path):
     )
 
 
+def chunks(seq, n):
```

Comment:
Well we need it in `writer.py` when we're writing to Redis. The alternative option would be to include it as a private function there, which I'm not terribly opposed to, but it kind of seems like in conflict with the existence of utilities like `absolute_path` which are equally generic.